imap_processing/ialirt/l0/parse_mag.py

```diff
@@ -691,10 +691,14 @@ def process_packet(
     )
 
     spherical = cartesian_to_spherical(gsm_vector)
+    if spherical.ndim == 1:
+        spherical = spherical[np.newaxis, :]
```

Comment:
Rather than special casing after the fact here, can you call `np.atleast_2d(mago_corrected)` earlier so you don't have to do both of these conditions?